### PR TITLE
RC1 Get Ready

### PR DESCRIPTION
**Pull request by marcgardiner**
So now, the first release of product got ready and this is just **release candidate 1**.

### What's involved in this version?
In this release, the following features are implemented:
- Reorganise the project so frontend is separated from backend
- Frontend was rebuilt in Angular so the app became SPA

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ require('./app_api/models/db');
 var uglifyJs = require("uglify-js");
 var fs = require('fs');
 
-var routes = require('./app_server/routes/index');
+// var routes = require('./app_server/routes/index');
```
Comment (igortalent) [inline app.js]: @marcgardiner why commented? any fair reason? why not remove?

File: app.js [hunk 2]
```diff
@@ -48,9 +51,13 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 app.use(express.static(path.join(__dirname, 'app_client')));
 
-app.use('/', routes);
+// app.use('/', routes);
```
Comment (igortalent) [inline app.js]: @marcgardiner same issue mentioned above

**Review by igortalent — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (marcgardiner) [reply app.js]: @igortb84 , that should be removed.

File: app.js (hunk 2 above)
Comment (marcgardiner) [reply app.js]: @igortb84 , same as above.

File: app.js (hunk 1 above)
Comment (marcgardiner) [reply app.js]: ok, just removed it

File: app.js (hunk 2 above)
Comment (marcgardiner) [reply app.js]: just fixed it with this commit: https://github.com/marcgardiner/zi-spa/pull/6/commits/240599d9cd73aefaf8aedb6e6a7ca425ce710fb4

**Review by igortalent — APPROVED** [review]
OK, @marcgardiner, good job! :)
Thanks.

**Comment by igortalent** [comment]
Feel free to merge now :)

**Pull request merged**